docs/pages/versions/unversioned/sdk/media-library.md

```diff
@@ -230,6 +240,7 @@ Subscribes for updates in user's media library.
 #### Arguments
 
 - **listener (_function_)** -- A callback that is called when any assets have been inserted or deleted from the library. **On Android** it's invoked with an empty object. **On iOS** it's invoked with an object that contains following keys:
+  - **shouldFullyReload (_boolean_)** -- Whether to all assets reloads is needed. When the user changes the permissions, this value will be set to `true` and you should reload all used assets.
```

Comment:
This name is very confusing without reading the whole docs and it suggest specific scenario that should be performed 😅 I'm not a fan of this 😉 

Let's see what is actually happening. The idea is to inform that the selection has changed yes? So what about naming it that way?
Names coming to my mind are: `assetsPermissionsChanged`, `permittedAssetsSelectionChanged` 🤔 

```suggestion
  - **assetsPermissionsChanged (_boolean_)** -- Flag indicating the permissions change in the `user's limited library selection`. If it's `true` this means that the user changed the access to the individual assets in the library. If this happens you should reload all used assets to resemble the actual access permissions.
```

cc @cruzach

Reply:
I agree, `assetsPermissionsChanged` makes it clear what's happening behind the scenes, rather than just us telling the user what to do (fully reload)
```suggestion
  - ** assetPermissionsChanged (_boolean_)** -- Flag indicating a permissions change occurred in the user's limited library selection. If `true`, the user has changed access to individual assets in their library, and you should reload all used assets to comply with the new permissions.
```